src/net.cpp

```diff
@@ -2933,3 +2932,27 @@ uint64_t CConnman::CalculateKeyedNetGroup(const CAddress& ad) const
 
     return GetDeterministicRandomizer(RANDOMIZER_ID_NETGROUP).Write(vchNetGroup.data(), vchNetGroup.size()).Finalize();
 }
+
+void CaptureMessage(const CAddress& addr, const std::string& msg_type, const Span<const unsigned char>& data, bool is_incoming)
+{
+    auto now = GetTime<std::chrono::microseconds>();
```

Comment:
nit: Maybe add a small comment here that this function captures the message at process time, not socket receive/send time. Nothing fancy, just a oneline comment to sum up https://github.com/bitcoin/bitcoin/pull/19509#issuecomment-765704796

Not only to document the expected behavior, but also to avoid people to "optimize" away this call to `GetTime` with the rationale that it doesn't matter and the message receive time can be used. See also https://github.com/bitcoin/bitcoin/pull/19509#discussion_r457076718